### PR TITLE
bot.ttl - added restrictions and annotations

### PR DESCRIPTION
**Pull request by wterkaj**
Contents of the revision:
1) Added the restrictions to bot:Building, bot:Space, bot:Storey
2) Added a few annotations in Italian

Other comments and open issues:
1) Domain of bot:hasElement to be considered for revision.
2) About the axiom "bot:Building owl:sameAs dbo:Building":
- owl:sameAs should be replaced by owl:equivalentClass, because owl:sameAs is used to link individuals
- it can be considered to move this alignment axiom to another ontology module, so to keep BOT ontology independent.
3) The individuals currently defined in the BOT ontology could be moved to a separate module that imports BOT.

### CODE REVIEW

File: bot/bot.ttl [hunk 1]
```diff
@@ -34,42 +34,73 @@
...
         rdfs:comment    "An independent unit of the built environment with a characteristic spatial structure, intended to serve at least one function or user activity [ISO 12006-2:2013]"@en ,
-						"Bauwerk hauptsächlich zum Zweck des Schutzes für seine Bewohner und die darin aufbewahrten Gegenstände; im Allgemeinen teilweise oder ganz geschlossen und ortsfest [ISO 6707-1:2014]"@de ,
-                        "En uafhængig del af det byggede miljø med en karakteristisk rumlig struktur, der understøtter mindst én funktion eller brugeraktivitet"@da .
+			"Bauwerk hauptsächlich zum Zweck des Schutzes für seine Bewohner und die darin aufbewahrten Gegenstände; im Allgemeinen teilweise oder ganz geschlossen und ortsfest [ISO 6707-1:2014]"@de ,
+                        "En uafhængig del af det byggede miljø med en karakteristisk rumlig struktur, der understøtter mindst én funktion eller brugeraktivitet"@da ;
+	rdfs:subClassOf 
+		[rdf:type owl:Restriction ;
+			owl:onProperty bot:hasElement ;
+			owl:allValuesFrom bot:Element
+		] ,
```
Comment (maximelefrancois86) [inline bot/bot.ttl]: the range of bot:hasElement is already bot:Element. This axiom adds no semantics

File: bot/bot.ttl [hunk 2]
```diff
@@ -34,42 +34,73 @@
...
+		[ rdf:type owl:Restriction ;
+			owl:onProperty bot:hasStorey ;
+			owl:allValuesFrom bot:Storey
+		] .
```
Comment (maximelefrancois86) [inline bot/bot.ttl]: the range of bot:hasStorey is already bot:Storey, This axiom adds not semantics

File: bot/bot.ttl [hunk 3]
```diff
@@ -34,42 +34,73 @@
...
         rdfs:comment    "A level part of a building"@en ,
-						"Die Gesamtheit aller Räume in einem Gebäude, die auf einer Zugangsebene liegen und horizontal verbunden sind"@de ,
-                        "Et plan i en bygning"@da .
+			"Die Gesamtheit aller Räume in einem Gebäude, die auf einer Zugangsebene liegen und horizontal verbunden sind"@de ,
+                        "Et plan i en bygning"@da ;
+	rdfs:subClassOf 
+		[ rdf:type owl:Restriction ;
+			owl:onProperty bot:hasElement ;
+			owl:allValuesFrom bot:Element
+		] ,
```
Comment (maximelefrancois86) [inline bot/bot.ttl]: same as above

File: bot/bot.ttl [hunk 4]
```diff
@@ -34,42 +34,73 @@
...
+		[ rdf:type owl:Restriction ;
+			owl:onProperty bot:hasSpace ;
+			owl:allValuesFrom bot:Space
+		] .
```
Comment (maximelefrancois86) [inline bot/bot.ttl]: same as above

File: bot/bot.ttl [hunk 5]
```diff
@@ -34,42 +34,73 @@
...
+                        "En afgrænset tredimensionel udstrækning defineret fysisk eller fiktivt"@da ;
+          rdfs:subClassOf 
+		[ rdf:type owl:Restriction ;
+			owl:onProperty bot:adjacentElement ;
+			owl:allValuesFrom bot:Element
+		] ,
+		[ rdf:type owl:Restriction ;
+			owl:onProperty bot:containsElement ;
+			owl:allValuesFrom bot:Element
+		] .
```
Comment (maximelefrancois86) [inline bot/bot.ttl]: bot:adjacentElement and bot:containsElement both have range bot:Element

**Comment by wterkaj** [comment]
Discarded.

**Comment by maximelefrancois86** [comment]
Can you do another commit just to delete the allValuesFrom axioms ?
I'd be happy to accept the italian labels and comments at least

Best,
Maxime

**Comment by pipauwel** [comment]
We have moved the BOT ontology (w3c-lbd-cg:bot-develop branch) to a separate repository: https://github.com/w3c-lbd-cg/bot. We have included the Italian labels there. So this is resolved.

**Pull request closed without merging**